### PR TITLE
Fix for issues #190 and #189.  Add missing street names to the __init…

### PR DESCRIPTION
**Pull request by tebrown**
Fix for issues #190 and #189.  Add missing street names to the __init__.py and add new training and unit tests to ensure exist tests pass. Change the unit test for `3121 187th street crt n east moline il 61244` so that it tags `187th street' as the StreetName and `crt` as the StreetNamePost.

As this changes an existing unit test, I think it warrants careful consideration before merging.

### CODE REVIEW

**Comment by jeancochrane** [comment]
Thanks for filing this @tebrown! I'll be able to review first thing tomorrow morning.

File: usaddress/__init__.py [hunk 1]
```diff
@@ -61,72 +61,81 @@
...
+    'traces', 'track', 'tracks', 'trafficway', 'trail', 'trailer', 'trails',
+    'trak', 'trce', 'trfy', 'trk', 'trks', 'trl', 'trlr', 'trlrs', 'trls',
+    'trnpk', 'trpk', 'trwy', 'tunel', 'tunl', 'tunls', 'tunnel', 'tunnels',
+    'tunnl', 'turn', 'turnpike', 'turnpk', 'un', 'underpass', 'union',
+    'unions', 'uns', 'upas', 'valley', 'valleys', 'vally', 'vdct', 'via',
+    'viadct', 'viaduct', 'view', 'views', 'vill', 'villag', 'village',
+    'villages', 'ville', 'villg', 'villiage', 'vis', 'vist', 'vista', 'vl',
+    'vlg', 'vlgs', 'vlly', 'vly', 'vlys', 'vst', 'vsta', 'vw', 'vws', 'walk',
+    'walks', 'wall', 'way', 'ways', 'well', 'wells', 'wl', 'wls', 'wy', 'xc',
+    'xg', 'xing', 'xrd', 'xrds'
```
Comment (jeancochrane) [inline usaddress/__init__.py]: flake8 flagged a trailing whitespace here, which is part of why [the Travis build failed.](https://travis-ci.org/datamade/usaddress/jobs/247236325)

File: usaddress/__init__.py [hunk 2]
```diff
@@ -61,72 +61,81 @@
...
+    'unions', 'uns', 'upas', 'valley', 'valleys', 'vally', 'vdct', 'via',
+    'viadct', 'viaduct', 'view', 'views', 'vill', 'villag', 'village',
+    'villages', 'ville', 'villg', 'villiage', 'vis', 'vist', 'vista', 'vl',
+    'vlg', 'vlgs', 'vlly', 'vly', 'vlys', 'vst', 'vsta', 'vw', 'vws', 'walk',
+    'walks', 'wall', 'way', 'ways', 'well', 'wells', 'wl', 'wls', 'wy', 'xc',
+    'xg', 'xing', 'xrd', 'xrds' 
 }
+
+
+
```
Comment (jeancochrane) [inline usaddress/__init__.py]: flake8 flagged too many blank lines (3) here, another reason the Travis build failed.

File: usaddress/__init__.py [hunk 3]
```diff
@@ -61,72 +61,81 @@
                   'northeast', 'northwest', 'southeast', 'southwest'])
 
 STREET_NAMES = {
-    'allee', 'alley', 'ally', 'aly', 'anex', 'annex', 'annx', 'anx', 'arc',
```
Comment (jeancochrane) [inline usaddress/__init__.py]: The diff is not super helpful here, so I pulled out what seem to be your new contributions:

```python
{'bl', 'causway', 'ci', 'ck', 'cr', 'crecent', 'cresent', 'crscnt',
'crssing', 'crt', 'ex', 'havn', 'height', 'hgts', 'la', 'lanes',
'lp', 'mi', 'mile', 'mn', 'overlook', 'ovlk', 'pk', 'plaines', 'prarie',
'pw', 'pwy', 'ri', 'rise', 'rn', 'strave', 'te', 'tl', 'tpk', 'tr', 'trpk',
'turn', 'xc', 'xg'}
```

Is that correct?

Comment (tebrown) [inline usaddress/__init__.py]: I apologize for this. I figured maintaining an alphabetical list was preferable without thinking about the diff.  This is the correct list

**Comment by tebrown** [comment]
Hi @jeancochrane. I have updated this merge request so that it passes the flake tests. The Python 2.7 test still failed, but this looked like it was an error downloading the requirements.

**Comment by tebrown** [comment]
Hi @jeancochrane, is there anything else you need from me?

File: usaddress/__init__.py (hunk 3 above)
Comment (jeancochrane) [inline usaddress/__init__.py]: No problem! Alphabetical is definitely preferable here – the diff confusion is git's fault, not yours. Looks good to me.

**Comment by jeancochrane** [comment]
Sorry for the delay @tebrown! I'm looking into the py2.7 error, which seems to be a Travis problem. Once that's done we'll be ready to merge.

**Comment by tebrown** [comment]
No problem! Thanks @jeancochrane!

**Pull request merged**